koku/reporting/provider/azure/models.py

```diff
@@ -117,6 +117,8 @@ class Meta:
 
     offer_id = models.PositiveIntegerField(null=True)
 
+    unit_of_measure = models.CharField(max_length=63, null=True)
```

Comment:
Since it seems like the unit is tied the meter information I was wondering if the unit of measure made sense to go in the `AzureMeter` model, where we're already keeping the corresponding rate.

Reply:
yeah lets do that